runtime/battery-station/src/parameters.rs

```diff
@@ -91,7 +92,9 @@ parameter_types! {
     pub const ContractsDepositPerByte: Balance = deposit(0,1);
     pub const ContractsDepositPerItem: Balance = deposit(1,0);
     pub const ContractsMaxCodeLen: u32 = 246 * 1024;
+    pub const ContractsMaxDebugBufferLen: u32 = 2 * 1024 * 1024;
     pub const ContractsMaxStorageKeyLen: u32 = 128;
+    pub const ContractsUnsafeUnstableInterface: bool = true;
```

Comment:
Can't find these two changes..

Reply:
Can you help me here?

Reply:
https://github.com/paritytech/substrate/blob/polkadot-v0.9.37/bin/node/runtime/src/lib.rs#L1219-L1220

Reply:
Obvious question: Why do we want to have the unsafe unstable interface?

Reply:
It enables experimental runtime calls for a whitelist of calls, which we want to utilize on Battery Station to offer developers to write dapps for prediction markets. We need this functionality especially in September for a Hackathon. It is only available as stable in v0.9.42. Production will never us unstable features.